src/game/Group/Group.cpp

```diff
@@ -1421,7 +1421,7 @@ bool Group::_addMember(ObjectGuid guid, char const* name, bool isAssistant, uint
                 player->m_InstanceValid = true;
     }
 
-    if (!isBGGroup())
+    if (!isBGGroup() && !(player && player->GetSession()->GetBot()))
```

Comment:
why is this needed?

Reply:
To prevent bot groups from being saved in database. Otherwise you get a whole bunch of startup errors next time about non-existent character in group.